test/built-ins/TypedArray/prototype/Symbol.toStringTag/this-is-not-object-strict-mode.js

```diff
@@ -0,0 +1,24 @@
+// Copyright (C) 2016 the V8 project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+es6id: 22.2.3.31
+description: Return undefined when `this` is null or undefined
+info: >
+  22.2.3.31 get %TypedArray%.prototype [ @@toStringTag ]
+
+  1. Let O be the this value.
+  2. If Type(O) is not Object, return undefined.
+  ...
+includes: [testTypedArray.js]
+flags: [onlyStrict]
```

Comment:
There are other tests like this in this PR. If we merge this file with the other one, we should do the same for other tests too.